app/src/main/java/com/example/footprnt/Discover/Adapters/ListAdapter.java

```diff
@@ -36,24 +34,26 @@ public ListAdapter(Context context, ArrayList<Business> businesses) {
         mContext = context;
     }
 
+
     @NonNull
     @Override
     public ListAdapter.BusinessViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
         View view = LayoutInflater.from(parent.getContext()).inflate(R.layout.item_business, parent, false);
-        return new BusinessViewHolder(view);
+        BusinessViewHolder viewHolder = new BusinessViewHolder(view);
```

Comment:
Why not simplify this to
  return new BusinessViewHolder(view);
??

Reply:
Checking my commit, I corrected it to "return new BusinessViewHolder (View)" However my commit was Reverted by Claire